src/test/java/edu/hm/hafner/analysis/ModuleDetectorTest.java

```diff
@@ -31,13 +31,18 @@
 
     private static final int NO_RESULT = 0;
 
-    private static final String PATH_PREFIX_MAVEN = "path/to/maven";
-    private static final String PATH_PREFIX_OSGI = "path/to/osgi";
-    private static final String PATH_PREFIX_ANT = "path/to/ant";
```

Comment:
Note, I separated these changes out into a separate commit, even though it's part of this PR. The problem was that `PATH_PREFIX_ANT + ModuleDetector.ANT_PROJECT` was actually returning `"path/to/antbuild.xml"`, which while that that technically works because the detector was using string search instead of path search, it's not _technically_ correct.

So I changed it so the path prefixes end with `/` so that the strings look like valid and expected paths.

If you'd like, I can split that off into a new branch so that the PR can be kept separate. But this is vital to work correctly for the build.gradle path detector, as without this fix, it was guessing that the module name should be `"to"` which is wrong.

Reply:
No this is fine. Actually I do not remember why it was in this way...